spring-cloud-dataflow-parent/pom.xml

```diff
@@ -64,6 +64,26 @@
 	</properties>
 	<dependencyManagement>
 		<dependencies>
+			<dependency>
+				<groupId>org.apache.tomcat.embed</groupId>
```

Comment:
I have already added this [here](https://github.com/spring-cloud/spring-cloud-dataflow/blob/9efd42b55ac8ade992a7b96c240ea1234b48f4cf/spring-cloud-dataflow-parent/pom.xml#L145-L171)